scripts/cmake/ClangSanitizer.cmake

```diff
@@ -0,0 +1,19 @@
+if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "3.6")
+	return()
+endif()
+
+option(OGS_ADDRESS_SANITIZER OFF "Use Clangs AddressSanitizer")
+option(OGS_UNDEFINED_BEHAVIOR_SANITIZER OFF "Use Clangs UndefinedBehaviorSanitizer")
+
+if(OGS_ADDRESS_SANITIZER)
+	set(SANITIZE_FLAG_VALUE "address")
+	set(ADDITIONAL_FLAGS "-fno-omit-frame-pointer")
+endif()
+
+if(OGS_UNDEFINED_BEHAVIOR_SANITIZER)
+	set(SANITIZE_FLAG_VALUE "${SANITIZE_FLAG_VALUE},undefined")
```

Comment:
[Docs](http://clang.llvm.org/docs/UsersManual.html#id33) state that:

> It is not possible to combine more than one of the -fsanitize=address, -fsanitize=thread, and -fsanitize=memory checkers in the same program.

So combining `undefined` with `address` should be ok.